web-ui/src/pages/streaming/inspection/[config]/[view].tsx

```diff
@@ -0,0 +1,163 @@
+// See the status of a input table or output view.
```

Comment:
What is the status of an input?
I think that documentation should be part of this PR.
There should be edits to the UI `tour` in docs.

Reply:
The documentation would also make it easier to review the code, since it's difficult to imagine how the result looks based on reading the code.

Reply:
I'm trying out https://www.meticulous.ai/ which (if it's any good) would be a way for us to do UI tests